### PR TITLE
refactor: GEOT-284: general styling part one

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: packages/ui/Navigation/index.tsx [hunk 1]
```diff
@@ -17,60 +18,71 @@ export const Navigation: FC<{
...
               label: 'My groups',
               icon: <IoIosKeypad />,
               url: '/my-groups',
             },
           ].map((value, index) => {
             return (
               <li className={styles.link} key={index}>
-                <Link href={value.url}>
-                  <a>
+                {value?.fn && (
```
Comment (ra8ga) [inline packages/ui/Navigation/index.tsx:60]: any better idea?

**Comment by mariusz-kabala** [comment]
```
./src/pages/account/styles.scss
SassError: Invalid CSS after "  }": expected selector or at-rule, was "}"
        on line 50 of /var/jenkins_home/workspace/(GTMS Frontend) Run checks on Pull Request/packages/app-andrew/src/pages/account/styles.scss
>>   }

   ---^
```

File: packages/app-andrew/src/components/group/GroupMembers/index.tsx [hunk 2]
```diff
@@ -11,6 +11,34 @@ export const GroupMembers: FC<{
   errorOccured: boolean
   users: IUser[]
 }> = ({ isLoading, errorOccured, users }) => {
+  const aaa = [
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/group/GroupMembers/index.tsx:14]: remove mocks

File: packages/app-andrew/src/pages/account/index.tsx [hunk 3]
```diff
@@ -155,26 +153,35 @@ export const AccountPage: NextPage<AccountPageProps> = () => {
...
-              name={state.name}
-              surname={state.surname}
-            />
-            <UserEmail email={state.email} additionalStyles={styles.userName} />
-            <DeleteAccount
-              additionalStyles={styles.deleteAccount}
-              onConfirm={() => null}
-            />
-          </div>
+          <p className={styles.desc}>
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/account/index.tsx:156]: there is already a component ```<UserDescription />``` remove mocks

File: packages/ui/Navigation/index.tsx [hunk 4]
```diff
@@ -17,60 +18,71 @@ export const Navigation: FC<{
...
-      )}
       <nav className={styles.navigation} data-testid="navigation">
+        {avatar && (
+          <UserAvatar additionalStyles={styles.avatar} image={avatar} />
+        )}
         <ul>
           {[
             {
-              id: 4,
+              id: 0,
```
Comment (mariusz-kabala) [inline packages/ui/Navigation/index.tsx:28]: why you have added id? removed it

File: packages/ui/Navigation/index.tsx (hunk 1 above)
Comment (mariusz-kabala) [reply packages/ui/Navigation/index.tsx:60]: yes, a lot, this is bullshit

File: packages/ui/NavigationDots/index.tsx [hunk 5]
```diff
@@ -3,31 +3,101 @@ import styles from './styles.scss'
...
 export const NavigationDots: FC<{ groups: IGroup[] }> = ({ groups }) => {
   if (groups.length === 0) {
     return null
   }
 
+  // this is just a mock
+  // @todo remove it when no longer needed
+  const links = [
+    {
+      id: 0,
```
Comment (mariusz-kabala) [inline packages/ui/NavigationDots/index.tsx:26]: why id is there?

File: packages/app-andrew/src/components/group/GroupMembers/styles.scss [hunk 6]
```diff
@@ -2,34 +2,45 @@
...
+    grid-template-columns: repeat(auto-fill, minmax($groupMembersAvatarSize, 1fr));
+    grid-gap: 15px;
+    
+    .avatar {
+      opacity: 0.5;
+    }
+
+    p {
+      position: absolute;
+      font-size: 16px;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/group/GroupMembers/styles.scss:42]: use the mixin for the font-size

File: packages/app-andrew/src/pages/account/index.tsx [hunk 7]
```diff
@@ -106,77 +106,65 @@ export const AccountPage: NextPage<AccountPageProps> = () => {
...
+            <Picture
+              {...(userQuery.hasAvatar('200x200')
+                ? userQuery.getAvatar('200x200')
+                : UserAvatarNoImage['200x200'])}
             />
-            <NotificationsSettings />
-          </div>
+          </a>
+          <UserDescription description={state.description} />
+          <TagsBar
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/account/index.tsx:134]: why tagsbar is here 2 times? looks like someone does not know how to merge...

File: packages/app-andrew/src/pages/group-create/styles.scss [hunk 8]
```diff
@@ -1,25 +1,26 @@
...
     padding: 0 40px;
-    background: $theme2;
 
     > div {
       width: 100%;
 
       h2 {
         font-size: 2.6em;
-        @include font-weight(bold);
+        @include font-weight(600);
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group-create/styles.scss:23]: use variable

File: packages/app-andrew/src/pages/group/[slug].tsx [hunk 9]
```diff
@@ -82,95 +85,136 @@ const GroupPage: NextPage<GroupPageProps> = (props) => {
...
-          </ErrorInfo>
-        )}
-        {state.notFound && <GroupNotFound />}
-        {state.hasNoAccess && <GroupNoAccess />}
-
-        {state.group && (
-          <div className={styles.columns}>
-            <div className={styles.column}>
+            <div>
+              <h2 data-tip={t('click-here-to-edit')} data-type="dark">
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/[slug].tsx:131]: React tooltip is in use here. Think if we need it, or you can do it in an other way

File: packages/ui/NavigationDots/index.tsx (hunk 5 above)
Comment (mariusz-kabala) [reply packages/ui/NavigationDots/index.tsx:26]: id is still here...

File: packages/ui/NavigationDots/styles.scss [hunk 10]
```diff
@@ -1,46 +1,76 @@
...
+
+      &:last-child {
+        margin-bottom: 0;
+      }
 
-    &:hover {
-      transition-delay: 200ms;
+      a {
+        display: block;
+        padding: 10px;;
```
Comment (mariusz-kabala) [inline packages/ui/NavigationDots/styles.scss:61]: double semicolon ;

File: packages/ui/NavigationTabs/styles.scss [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+    @include font-weight(semiBold);
+  }
+
+  ul {
+    display: flex;
+    margin-left: 20px;
+
+    li {
+      margin-right: 10px;
+      font-size: 16px;
```
Comment (mariusz-kabala) [inline packages/ui/NavigationTabs/styles.scss:22]: use mix-in

File: packages/ui/PostSingle/PostResponse/index.tsx [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+      data-testid="post-single"
+    >
+      <div className={styles.header}>
+        <Link href={`/user/${owner}`}>
+          <div className={styles.user}>
+            <UserAvatar
+              image={noImage['35x35']}
+              additionalStyles={styles.userAvatar}
+            />
+            {/* @mariusz use this
```
Comment (mariusz-kabala) [inline packages/ui/PostSingle/PostResponse/index.tsx:31]: use for what?

File: packages/ui/PromotedTags/index.tsx [hunk 13]
```diff
@@ -59,11 +79,22 @@ export const PromotedTags: FC<{
...
         </div>
       )}
+
       {tags.length === 0 && isAdmin && (
-        <p>
-          This group has no promoted tags{' '}
-          <a onClick={onClick}>please add some</a>
-        </p>
+        <div className={styles.mock}>
+          <div />
```
Comment (mariusz-kabala) [inline packages/ui/PromotedTags/index.tsx:85]: wtf are those divs?

Comment (ra8ga) [reply packages/ui/PromotedTags/index.tsx:85]: this is just temporary mock

Comment (ra8ga) [reply packages/ui/PromotedTags/index.tsx:85]: removed

File: packages/app-andrew/src/pages/account/index.tsx (hunk 7 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/account/index.tsx:134]: removed

File: packages/app-andrew/src/pages/group-create/styles.scss (hunk 8 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group-create/styles.scss:24]: fixed

File: packages/app-andrew/src/pages/group/[slug].tsx (hunk 9 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group/[slug].tsx:131]: it will be removed in next PR

**Pull request merged**